package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@sinet/pathlint",
+  "version": "1.0.0",
+  "description": "Enforces specific file/directory name patterns for uniformity. It matches files/directories in the current working directory. Great for projects involving several developers.",
+  "author": {
+    "name": "Vincent John Cagay",
+    "email": "vjcagay@gmail.com"
+  },
+  "contributors": [
+    {
+      "name": "Daniel Arbiol",
+      "email": "arbiol.daniel0727@gmail.com"
+    },
+    {
+      "name": "Dedito Cantila",
+      "email": "dedito.cantila@gmail.com"
+    },
+    {
+      "name": "Vincent John Cagay",
+      "email": "vjcagay@gmail.com"
+    }
+  ],
+  "keywords": [
+    "lint",
+    "linting",
+    "filename",
+    "file-name",
+    "file name",
+    "directory",
+    "directoryname",
+    "directory name",
+    "foldername",
+    "folder-name",
+    "folder name",
+    "path",
+    "pathlint"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/School-Improvement-Network/pathlint.git"
+  },
+  "bugs": {
+    "url": "https://github.com/School-Improvement-Network/pathlint/issues"
+  },
+  "license": "MIT",
+  "main": "./lib/pathlint.js",
+  "scripts": {
+    "test": "npm run eslint -- --reset --eslintrc ./ && npm run jscs -- ./ && npm run pathlint && gulp test",
```

Comment:
Should test run the linting stuff, or should it be separated out? Test seems like it should just run tests to me. On the flip side maybe we should add a convenience `npm run lint` that would run all the linting (eslint, jscs, pathlint).